internal/generate/testdata/clusterserviceversions/output/memcached-operator-multiVersion.yaml

```diff
@@ -259,4 +261,4 @@ spec:
     sideEffects: None
     targetPort: 9443
     type: ValidatingAdmissionWebhook
-    webhookPath: /validate-cache-example-com-v1alpha1-memcached
\ No newline at end of file
+    webhookPath: /validate-cache-example-com-v1alpha1-memcached
```

Comment:
Curious, why does the `webhook` path get added? `Webhookdefinitions` are defined in operator-framework/api

Reply:
It only added a newline at the end of the file. Same as with https://github.com/operator-framework/operator-sdk/pull/5533/files#diff-bd889e1aa1c1539a692594a301ce58562dd1f903280a9c0a324d8503a2f8b1eaR212.

Reply:
Sorry, my bad. Overlooked it.